frontend/src/components/SigninPage.js

```diff
@@ -71,7 +76,12 @@ function SigninPage() {
                 sx={{marginBottom: "2rem"}}
             />
             <Box display={"flex"} flexDirection={"row"} justifyContent={"flex-end"}>
-                <Button variant={"outlined"} onClick={handleSignin}>Sign in</Button>
+                <Button variant={"outlined"} onClick={handleSignin}>Submit</Button>
+            </Box>
+
+
+            <Box display={"flex"} flexDirection={"row"} justifyContent={"flex-end"}>
+                <Button variant={"outlined"} component={Link} to="/signup">Sign Up</Button>
```

Comment:
Would it be better to use the constant defined in configs file?

Reply:
fixed